rest_server/config.py

```diff
@@ -0,0 +1,66 @@
+"""Config settings."""
+
+import dataclasses as dc
+import logging
+
+import coloredlogs  # type: ignore[import]
+from wipac_dev_tools import from_environment_as_dataclass
+
+LOGGER = logging.getLogger("skydriver")
+
+# --------------------------------------------------------------------------------------
+# Constants
+
+
+@dc.dataclass(frozen=True)
+class EnvConfig:
+    """Environment variables."""
+
+    # pylint:disable=invalid-name
+    AUTH_AUDIENCE: str = "skydriver"
+    AUTH_OPENID_URL: str = ""
+    MONGODB_AUTH_PASS: str = ""  # empty means no authentication required
+    MONGODB_AUTH_USER: str = ""  # None means required to specify
+    MONGODB_HOST: str = "localhost"
+    MONGODB_PORT: int = 27017
+    REST_HOST: str = "localhost"
+    REST_PORT: int = 8080
+    CI_TEST: bool = False
+
+
+ENV = from_environment_as_dataclass(EnvConfig)
+
+
+USER_ACCT = "skydriver-service-account"
+SKYMAP_SCANNER_ACCT = "skymap-scanner-service-account"
+
+EXCLUDE_DBS = [
+    "system.indexes",
+    "production",
+    "local",
+    "config",
+    "token_service",
+    "admin",
+]
+
+EXCLUDE_COLLECTIONS = ["system.indexes"]
```

Comment:
What are these two constants used for?  They don't seem to show up in the code.

Reply:
That's copypasta from the mou dash